Tests/AblyChatTests/IntegrationTests.swift

```diff
@@ -82,25 +96,99 @@ struct IntegrationTests {
         let rxReactionFromSubscription = try #require(await rxReactionSubscription.first { _ in true })
         #expect(rxReactionFromSubscription.type == "heart")
 
+        // MARK: - Occupancy
+
+        // It can take a moment for the occupancy to update from the clients connecting above, so we’ll wait a 2 seconds here.
+        try await Task.sleep(nanoseconds: 2_000_000_000)
+
+        // (12) Get current occupancy
+        let currentOccupancy = try await rxRoom.occupancy.get()
+        #expect(currentOccupancy.connections != 0) // this flucuates dependant on the number of clients connected e.g. simulators running the test, hence why checking for non-zero
```

Comment:
I think you need to give a unique name for the room and check exact numbers tbh.

Reply:
it's a bit flaky because you could have X simulators open (like I did) and receive an entirely different number. We can revisit this as part of the ticket in the description to improve tests... I want to get as many features done before friday